### PR TITLE
Allison i

### PR DESCRIPTION
**Pull request by allisoninman**
I changed the wording of the authorship but you still get the credit!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # How to Git Better at Git
 
-- Daric Teske
+- Allison Inman didn't write this, Daric Teske did
```
Comment (gauriramesh) [inline README.md]: You forgot a period at the end of your sentence, pls change otherwise Amy Cyphers will be mad.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # How to Git Better at Git
 
-- Daric Teske
+
+- Allison Inman didn't write this, Daric Teske did
```
Comment (troynguyen8) [inline README.md:4]: Was this change really necessary?

**Comment by troynguyen8** [comment]
I added a comment about the necessity of a change. Overall the changes look great though 👍

**Review by dteske25 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (allisoninman) [reply README.md]: Plz don't tell her I am so scared
